snmplib/snmp-tc.c

```diff
@@ -255,7 +255,7 @@ ctime_to_timet(const char *str)
      */
 
 #ifdef HAVE_STRUCT_TM_TM_ISDST
-#if HAVE_DECL_DAYLIGHT
```

Comment:
This change seems incorrect to me. I'm concerned this change will cause a compiler error if HAVE_DECL_DAYLIGHT is not defined.